contracts/src/test/kotlin/com/r3/corda/lib/tokens/contracts/ContractTestCommon.kt

```diff
@@ -68,12 +68,12 @@ abstract class ContractTestCommon {
 
 /**
  * Creates a [NonFungibleToken] from an [IssuedTokenType].
- * E.g. IssuedTokenType<TokenType> -> NonFungibleToken<TokenType>.
+ * E.g. IssuedTokenType<TokenType> -> NonFungibleToken.
  * This function must exist outside of the contracts module as creating a unique identifier is non deterministic.
  */
-infix fun <T : TokenType> IssuedTokenType<T>.heldBy(owner: AbstractParty): NonFungibleToken<T> = _heldBy(owner)
+infix fun IssuedTokenType.heldBy(owner: AbstractParty): NonFungibleToken = _heldBy(owner)
```

Comment:
You have 2 functions that do the same now, but only takes Party and other AbstractParty. Why? see: https://github.com/corda/token-sdk/pull/110/files#diff-69f95b21984486950778649bdf50c5b0R66

Reply:
Chris Rankin moved these. We need them here because they cannot be defined inside the `contracts` main module. They are not part of the public API so we can find a better solution at some point in the future.

Reply:
and yet in TokenUtilties we have...
`infix fun IssuedTokenType.heldBy(party: Party): NonFungibleToken {
    return NonFungibleToken(this, party, UniqueIdentifier())
} `